### DISCUSSION TITLE
Уведомления push на определенные события

### DISCUSSION THREAD

**Discussion by sbca68** [rosteleset/SmartYard-Server#33] [Ideas]
добавить возможность настроить получение уведомления о событии для пользователя.
типы событий:
открытие ключом,
открытие по лицу,
открытие по коду,
открытие из приложения,
открытие по звонку

**Comment by sbca68** [comment]
как выбрать по какому лицу отправлять уведомление, если у одного человека их много добавлено?

**Reply by kulakoff** [reply]
Создавать горуппы (person_id, person_name),  в приложении пользователь сам указывает person_name, например "Брат", "Брат 2", "Бабушка" 
- у пользователя может быть зарегистрированно несколько лиц, можно  добавить разумные ограничения для зарегистрированных лиц на пользователя.
-  добавить ключ кользователя
- включить push уведомления для этого пользователя

**Reply by kulakoff** [reply]
Добавить ограничения на количество person_id в квартире, базовай тариф, расширенный
